.github/workflows/deploy_docs.yml

```diff
@@ -31,6 +31,8 @@ jobs:
         working-directory: docs
         
       - name: Deploy
-        run: npm run deploy-gh-pages
-        working-directory: docs
+        uses: JamesIves/github-pages-deploy-action@v4.3.0
```

Comment:
Hopefully this remains stable?

Reply:
That repo had many stars, so I guess it will be.